tests/0134-purge_same_partition_messages_after_seek_in_same_batch_consume.c

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
Following the repo style, this filename is way too long and is more of a test description (which goes in the test itself).
Suggest renaming to 0136-toppar_barrier.c.
Use number 0136:  134 and 135 are taken in other PRs.